src/main/java/mekanism/client/render/RenderTickHandler.java

```diff
@@ -304,6 +311,25 @@ public void onBlockHover(DrawSelectionEvent.HighlightBlock event) {
             MultiBufferSource renderer = event.getMultiBufferSource();
             Camera info = event.getCamera();
             PoseStack matrix = event.getPoseStack();
+
+            ItemStack stack = player.getMainHandItem();
+
+            // Draw outlines for area mining blocks
+            if (!outliningArea && !player.isShiftKeyDown() && stack.getItem() instanceof ItemMekaTool tool && tool.isModuleEnabled(stack, MekanismModules.BLASTING_UNIT)) {
+                outliningArea = true;
+                Vec3 renderView = info.getPosition();
+
+                Set<BlockPos> blocks = ModuleBlastingUnit.findPositions(world, pos, rayTraceResult.getDirection(), tool.getModule(stack, MekanismModules.BLASTING_UNIT).getCustomInstance().getBlastRadius());
```

Comment:
It will require one extra level of if statements but it is probably worth putting `tool.getModule(...)` in a local variable and validating that it isn't null and is enabled rather than using the helper `isModuleEnabled` method so that we can avoid having to lookup/load the module from NBT twice.